dev_guide/routes.adoc

```diff
@@ -121,7 +148,47 @@ remove a passphrase from a keyfile, you can run:
 ----
 # openssl rsa -in <passwordProtectedKey.key> -out <new.key>
 ----
+endif::[]
+ifdef::openshift-online[]
+Unsecured routes are the default configuration, and are therefore the simplest
+to set up.
+However,
+xref:../architecture/core_concepts/routes.adoc#secured-routes[secured routes]
+offer security for connections to remain private.  To create a secured HTTPS route
+encrypted with the default certificate for {product-title} {product-version}
+you can use the `create route` command.
```

Comment:
Only the last sentence of the paragraph varies between Online and non-Online, so the ifdeffing could be still a little more fine-grained, but I don't feel strongly about it.